app/src/components/UserDetailsHeader.tsx

```diff
@@ -53,6 +61,9 @@ const UserDetailsHeader: React.FC = () => {
         <Text onPress={() => navigation.navigate("PGP")}>
           Import your PGP/GPG key pair
         </Text>
+        <Text onPress={showEthAddress}>
+          Wallet Address: {ethAddress ? truncateEthAddress(ethAddress) : ""}
```

Comment:
No need to truncate unless it wraps...

```suggestion
          {ethAddress}
```